estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">estate.property.view.search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Search Properties">
+                <field name="name"/>
+                <field name="postcode"/>
+                <field name="expected_price"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="facades"/>
+                <field name="tags"/>
+
+                <filter string="Status" name="status" domain="['|', ('state', '=', 'new'), ('state', '=', 'offer_received')]"/>
+                <filter string="High Value" name="high_value" domain="[('expected_price', '&gt;', 1000000)]"/>
+                <filter string="Low Value" name="low_value" domain="[('expected_price', '&lt;', 1000000)]"/>
+                <separator/>
+                <filter string="Include archived" name="archived" domain="['|', ('active', '=', True), ('active', '=', False)]"/>
+                <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
+
+                <group expand="0" string="Group By">
+                    <filter string="Active" name="active" domain="['|', ('active', '=', True), ('active', '=', False)]" context="{'group_by': 'active'}"/>
+                    <filter string="Postcode" name="postcode" domain="[]" context="{'group_by': 'postcode'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="estate_property_view_form" model="ir.ui.view">
+        <field name="name">estate.property.view.form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Property">
+                <sheet>
+                    <widget name="web_ribbon" title="Archived" bg_color="bg-danger" invisible="(active)"/>
```

Comment:
No parenthesis inside invisible attribute :wink: There are a bunch of other similar issues in this file, please address those too :+1: